lib/LaTeXML/Core/State.pm

```diff
@@ -422,6 +422,26 @@ sub lookupExpandable {
     return $defn; }
   return; }
 
+# Whether token must be wrapped as dont_expand
+sub isDontExpandable {
+  my ($self, $token) = @_;
+  # Basically: a CS or Active token that is either not defined, or is expandable
+  # (but not \let to a token)
+  return unless $token;
+  my $defn;
+  my $entry;
+  #  my $inmath = $self->lookupValue('IN_MATH');
+  my $cc = $$token[1];
+  if ($CATCODE_ACTIVE_OR_CS[$cc]) {
```

Comment:
btw this `$CATCODE_ACTIVE_OR_CS` check is redundant with the outer check from don't expand for
`(($cc == CC_CS) || ($cc == CC_ACTIVE))` . Maybe one of the two should be phased out?